test/e2e/ginkgo/README.md

```diff
@@ -0,0 +1,25 @@
+# Kubestellar end to end testing
+The end to end testing includes:
+1. deployments are downsync propegated to the WECs.
+1. Update of the workload object on WDS should update the object on the WECs. Increase the number of replicas from 1 to 2, verify they are updated on the WECs.
+1. Changing the bindingpolicy objectSelector to no longer match should delete the object from the WECs.
+1. Changing the bindingpolicy objectSelector to match should create the object on the WECs.
+1. Delete of the bindingpolicy object should delete the object on the WECs.
+1. Delete of the overlapping bindingpolicy object should not delete the object on the WECs.
```

Comment:
What overlapping BindingPolicy? This was not mentioned before.

Reply:
This previously existed in the update-delete.sh test.  By overlapping bindingpolicy it is meant a binding policy that results in the same objects downsynced into the same clusters.

Reply:
@MikeSpreitzer this is a special case of 2 BPs on the same WDS that update the same object going to the same WEC.  In OCM one of them will "win"  but when we remove one of the BP the object will move to be "owned" by the other one and shouldn't be deleted.

Reply:
@ezrasilvera : as @effi-ofer mentioned, there is no "win" here, it is a matter of redundancy.

@effi-ofer: My point here is about the language in this README. Where it refers to "the overlapping bindingpolicy", an English reader expects to have already been told about an "overlapping bindinbpolicy". OTOH, if this text said "an overlapping bindingpolicy" then that expectation would not be there.

Reply:
@MikeSpreitzer It's not redundancy it is a  win in term of ownership.  As a result from the overlapping BP two manifestWorks are created. However on the WEC the OCM agent will treat only one of them as the source for that object so both manifestwork will fight on the ownership.   Once one of the binding is removed --> one of the manifestWork is removed --> the OCM agent will change the ownership to the other manifestwork
In our case it's not so interesting as we are on the same WDS --> same object --> it doesn't matter who wins. But if we do the same test with different WDS and create the same object (name, NS, etc..)  with couple of different fields then the question of "winning ownership" has more meaning.

Reply:
@ezrasilvera : sorry, I misread your statement, did not notice the "in OCM" qualification. However, it is not a matter of one BindingPolicy or another winning, the OCM question is about one ManifestWork or another winning. I think that the KubeStellar user does not care about this question, since overlapping BindingPolicies in one WDS will produce redundant content in the ManifestWorks, so it will not matter which one OCM marks as object owner in the WEC.

Reply:
I am sorry but I am getting lost with all the explanations :( Is the problem the use of "the" rather than "an"?  If so I agree, and will change.